### ISSUE TITLE
[Bug]: Form Button component is not working properly

### ISSUE THREAD

**Issue by tiodot** [lowcoder-org/lowcoder#1036]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

In the builder, insert a Form or Form Button,  the button will show error message

![image](https://github.com/user-attachments/assets/94613ec6-8d2e-4f68-9169-2faba36c2528)

### Expected Behavior

Form Button works properly

### Steps to reproduce

1. Create or open a app.
2. Insert Form component or Form Button component

### Environment

The official website

**Comment by DanielOaks** [comment]
Hey there! I just migrated from Openblocks to Lowcoder yesterday, and I'm getting the same issue with a slightly different error message (clicking to reload doesn't change things). I did manage to get it working on one of my other apps by continuously deleting and remaking the form button several times, somehow, but that hasn't been super consistent.

<img width="576" alt="Screenshot 2024-07-13 at 2 32 51 pm" src="https://github.com/user-attachments/assets/f07155d5-e4eb-4736-b098-ec1e6641b206">
